### ISSUE TITLE
result of the code return format

### ISSUE THREAD

**Issue by Gsuhy-L** [ltian678/DUCK-code#4]
the coding result of return contain 2 parm:

 emb, out_labels= model(Batch_data), 

but when i looking the model of SimpleGATBERTNet,
        x = self.fc1(gnn_x)
       x = self.fc2(x)
        x = F.log_softmax(x, dim=1)
        return x
this only return one parm， Am I wrong

**Comment by joshchang0111** [comment]
Hey @Gsuhy-L , 

I also encounter the same problem few weeks ago, the code provided by the author has several problems.
For the issue you mentioned here, modifying the following line
```
emb, out_labels = model(Batch_data)
```
to
```
out_labels = model(Batch_data)
```
solves the problem.
